work_queue/src/work_queue_json.c

```diff
@@ -44,7 +44,7 @@ static const char *work_queue_task_properties[] = { "tag", "command_line", "work
 	"time_execute_cmd_finish", "total_transfer_time", "cmd_execution_time",
 	"total_cmd_execution_time", "total_cmd_exhausted_execute_time",
 	"total_time_until_worker_failure", "total_bytes_received", "total_bytes_sent",
-	"total_bytes_transferred", "time_app_delay", NULL
+	"total_bytes_transferred", "time_app_delay", "cores", "memory", "disk", NULL
```

Comment:
We may want to transform this array into a hash_table in the future.

Reply:
Yes, validating incoming json will require some care in the future...